core/modules/system/system.module

```diff
@@ -3397,12 +3397,19 @@ function system_header_block($settings) {
  * @see system_block_view()
  */
 function system_breadcrumb_block($settings = array()) {
+  $output = '';
   $crumbs = backdrop_get_breadcrumb();
-  if (isset($settings['current']) && $settings['current'] == TRUE) {
-    $crumbs[] = '<span aria-current="page">' . backdrop_get_title() . '</span>';
+  if (!empty($crumbs)) {
+    if (empty($settings['current'])) {
+      // Hide the last link if the block settings don't call for showing it, for
+      // backwards compatibility.
+      $last_crumb = array_pop($crumbs);
+      $crumbs[] = str_replace('class="', 'class="element-invisible ', $last_crumb);
```

Comment:
Is there an other, more "standard" PHP or Backdrop/Drupal way to add the class here?